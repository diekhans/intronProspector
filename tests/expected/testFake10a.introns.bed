fake10	10026	489051	sj0_GC/AG	1	+
fake10	34666	35513	sj1_GT/AG	1	-
fake10	130577	131433	sj2_GT/AG	1	+
fake10	131550	132801	sj3_GT/AG	1	+
fake10	132879	133326	sj4_GT/AG	1	+
fake10	133449	136100	sj5_GT/AG	1	+
fake10	136650	139594	sj6_GT/AG	1	+
fake10	487560	488264	sj7_GT/AG	37	+
fake10	488522	488947	sj8_GT/AG	46	+
fake10	489118	490086	sj9_GT/AG	44	+
