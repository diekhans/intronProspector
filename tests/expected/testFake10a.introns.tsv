chrom	intron_start	intron_end	strand	uniq_mapped_count	multi_mapped_count	unsure_mapped_count	max_left_overhang	max_right_overhang	confidence	splice_sites
fake10	10026	489051	+	0	1	0	14	67	0	GC/AG
fake10	34666	35513	-	0	1	0	81	20	0	GT/AG
fake10	130577	131433	+	0	1	0	72	29	0	GT/AG
fake10	131550	132801	+	0	1	0	85	16	0	GT/AG
fake10	132879	133326	+	1	0	0	80	17	0	GT/AG
fake10	133449	136100	+	0	1	0	15	85	0	GT/AG
fake10	136650	139594	+	0	1	0	75	26	0	GT/AG
fake10	487560	488264	+	37	0	0	68	90	3.92	GT/AG
fake10	488522	488947	+	44	2	0	87	91	4.47	GT/AG
fake10	489118	490086	+	42	2	0	93	90	4.85	GT/AG
